templates/zerver/emails/realm_deactivated.source.html

```diff
@@ -0,0 +1,22 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/registration_confirmation.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+<p>
+    {% if acting_user_mail %}
```

Comment:
maybe renaming it to:
`if send_to_deactivating_owner` makes more sense?

not sure though :)

Reply:
Yeah, we could rename it though.